mkt/site/middleware.py

```diff
@@ -169,9 +169,10 @@ class DeviceDetectionMiddleware(object):
     def process_request(self, request):
         dev = request.GET.get('dev')
         if dev:
-            setattr(request, 'MOBILE', dev == 'android')
-            setattr(request, 'GAIA', dev == 'firefoxos')
-            setattr(request, 'TABLET', dev == 'desktop')
+            request.MOBILE = dev == 'android'
```

Comment:
We should remove or rename that `MOBILE` constant, it's confusing. Hell, of those constants are becoming super confusing, `GAIA` doesn't mean much, especially if it's false for tvs.